webpack/assets/javascripts/foreman_tools.js

```diff
@@ -1,3 +1,7 @@
+'use strict';
+
+const $ = require('jquery');
```

Comment:
@tbrisker I think we should consider declaring dependencies explicitly instead of relying on the webpack ProvidePlugin for our modules. Relying on the plugin creates a strange dependency on the build system. We wouldn't be able to export and package the code for reuse without explicitly declaring the dependencies.

I think leaving the plugin in place to as a safety net for legacy code that might be moved to webpack assets without being refactored makes sense.